nvdaHelper/localWin10/sconscript

```diff
@@ -44,7 +41,8 @@ localWin10Lib = env.SharedLibrary(
 # Therefore, we must include it.
 # VS 2017 keeps changing the path to reflect the latest major.minor.build version which we canot easily find out.
 # Therefore  Search these versioned directories from newest to oldest  to collect all the files we need.
-vcRedistDirs = glob.glob(os.path.join(progFilesX86, r"Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.1*\x86\Microsoft.VC141.CRT"))
+vcProductDir = find_vc_pdir("14.1")

```

Comment:
Can you please fetch the vc version from env.get('MSVC_VERSION') and use that rather than hardcoding 14.1 on this line and the next line also. Otherwise, we may forget to update the version string here if we ever move to a later version of Visual Studio.

Reply:
Done.
Please note `Microsoft.VC141.CRT`, `msvcp140.dll`, vccorlib140.dll` and `vcruntime140.dll` still hold version numbers in their very names, but I doubt there is much we can do except maybe externalizing these strings.
Appart from these, the only remaining hard-coded reference to "14.1" is in `nvdaHelper/archBuild_sconscript` in which we check if the latest available version meets our requirement rather than instructing which version we need. This is safe only as long as we do not pull a newer SCons supporting a newer MSVS.